### PR TITLE
feat: [IDP-2561]: Updated the Onboarding guide with V2 onboarding steps

### PR DESCRIPTION
**Pull request by Debanitrkl**
## Description

* Please describe your changes: Updated the Onboarding guide with V2 onboarding steps_

## PR lifecycle

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/internal-developer-portal/get-started/setup-git-integration.md [hunk 1]
```diff
@@ -13,43 +13,25 @@ This guide describes the steps a Harness account admin can take to set up the ID
...
-
-1. User lands on IDP module by navigating from sidebar after logging into their Harness account.
-
-![](static/option1.png)
-![](static/option2.png)
-
-2. If the required prerequisite is full-filled, user can proceed with the harness catalog onboarding. Else user will be at the same page till the prerequisite is full-filled.
+- Only users who are **Harness Account Admin** or assigned with **IDP Admin** role having the adequate permission for the as displayed below can perform the onboarding. We recommend creating an **IDP Admin** role with the below mentioned permission and assign it to your team who will be responsible for configuring IDP. Here's the detailed [documentation on how to assign roles](https://developer.harness.io/docs/platform/role-based-access-control/add-user-groups/#assign-roles-and-resource-groups) 
 
-3. Only Harness account admins and users having the "Manage" permission for the "Developer Portal" resource can perform the onboarding. We recommend creating an "IDP Admin" role with the above mentioned permission and assign it to your team who will be responsible for configuring IDP.
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/get-started/setup-git-integration.md:25]: IDP Admin role is auto-created I believe. No?

**Review by OrkoHunter — APPROVED** [review]

File: docs/internal-developer-portal/get-started/setup-git-integration.md (hunk 1 above)
Comment (Debanitrkl) [reply docs/internal-developer-portal/get-started/setup-git-integration.md:25]: @sathish-soundarapandian need a confirmation on this.

Comment (sathish-soundarapandian) [reply docs/internal-developer-portal/get-started/setup-git-integration.md:25]: Yes. It is auto created.

**Comment by bhavya-sinha** [comment]
@Debanitrkl Can we add the screenshot from the UI flow, instead of figma mocks?

**Review by bhavya-sinha — DISMISSED** [review]

**Comment by Debanitrkl** [comment]
> @Debanitrkl Can we add the screenshot from the UI flow, instead of figma mocks?

sure will update.

**Review by bhavya-sinha — APPROVED** [review]

**Review by sathish-soundarapandian — APPROVED** [review]

**Pull request merged**